src/components/list/ListHeader.tsx

```diff
@@ -0,0 +1,11 @@
+import { css } from '@patternfly/react-styles';
+import React, { ReactNode } from 'react';
+import { styles } from './header.styles';
+
+interface ListHeaderProps {
+  children: ReactNode;
+}
+
+export const ListHeader: React.SFC<ListHeaderProps> = ({ children }) => (
+  <header className={css(styles.header)}>{children}</header>
+);
```

Comment:
I don't know that we should componitize the list as AWS and OCP may not have the exact layout in the future? 

Considering the PF4 data list component should be ready after the next PF4 sprint, perhaps we should wait to update that, first?